product-service/lambda_func/utils/createProduct.ts

```diff
@@ -0,0 +1,52 @@
+import { DynamoDB } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocument } from '@aws-sdk/lib-dynamodb';
+
+type Product = {
+    id: string,
+    title: string,
+    description: string,
+    count: number,
+    price: number
+}
+
+function generateUUIDv4() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+        var r = Math.random() * 16 | 0,
+            v = c === 'x' ? r : (r & 0x3 | 0x8);
+        return v.toString(16);
+    });
+}
```

Comment:
You can use the `randomUUID()` function from the `node:crypto` module
Documentation: https://nodejs.org/docs/latest/api/crypto.html#cryptorandomuuidoptions